PlayerUI/PlayerUICustomization/objc/PlayerUICustomization/ViewController.m

```diff
@@ -219,14 +219,13 @@ - (IBAction)setNextLayout:(id)sender
             
             // Alternatively you can customize a single-state button
             // with an image instead
-            /*
```

Comment:
I like having the code un-commented. It makes this customization hard to miss even for someone who's casually browsing through the sample app on their device or simulator.